packages/core/src/components/Accordion/AccordionItem/AccordionItem.tsx

```diff
@@ -5,28 +5,53 @@ import { VibeComponentProps, ElementContent } from "../../../types";
 
 export interface AccordionItemProps extends VibeComponentProps {
   /**
-   * Header title
+   * Header title for the accordion item.
    */
   title?: ElementContent;
   /**
    * The value of the expandable section
    */
   children?: ElementContent;
   /**
-   * The expand icon font size
+   * The font size of the expand/collapse icon.
    */
   iconSize?: number | string;
   /**
-   * On click callback
+   * Custom callback triggered when the item is clicked.
    */
   onClick?: () => void;
+  /** @ignore */
   open?: boolean;
+  /** @ignore */
   onClickAccordionCallback?: () => void;
+  /**
+   * Determines whether the item's border is hidden.
+   */
   hideBorder?: boolean;
+  /**
+   * Custom class name to add to the header of the expandable
+   */
   headerClassName?: string;
+  /**
+   * Custom class name to add to the content of the expandable
+   */
   contentClassName?: string;
+  /**
+   * Custom class name to add to the ExpandCollapse component
+   */
   expandCollapseComponentClassName?: string;
+  /**
+   * Determines whether click events are captured.
```

Comment:
maybe we should expand a little more on that, saying capture phase or expand it even more, I was surprised to know not all devs know what capture is